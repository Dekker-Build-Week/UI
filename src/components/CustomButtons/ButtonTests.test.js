import React from 'react';

import Button from './Button';
import renderer from 'react-test-renderer';

describe('Button test suite', () => {
    it('Renders the Button correctly', () => {
        const tree = renderer
            .create(<Button><p>Test</p></Button>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    })
});