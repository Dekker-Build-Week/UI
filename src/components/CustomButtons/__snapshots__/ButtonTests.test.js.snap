// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test suite Renders the Button correctly 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root makeStyles-button-1 MuiButton-text"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <p>
      Test
    </p>
  </span>
</button>
`;
